src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,37 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <>
```

Comment:
Fragment is redundant here cause you have div CommentInfo, which is parent for other elements